.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Dev CI/CD
+
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, synchronize]
+env:
+  REACT_APP_INFURA_ID: ${{ secrets.DEV_INFURA_ID }}
+  GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
```

Comment:
Neither of these environment variables is necessary.

Reply:
👍  Removed